### PR TITLE
More ME32 fixes

### PR DESCRIPTION
**Pull request by simonwo**
- ME32: Cut down on the number of test cases
- ME32: Handle dead fields as well as null cases.
- ME32: Prevent trigger on measure deletion.
- Measure business rules formatting.
- ME32: Use current indents to ignore past history of codes

### CODE REVIEW

File: measures/business_rules.py [hunk 1]
```diff
@@ -297,28 +302,47 @@ def validate(self, measure):
...
+            query &= Q(dead_additional_code=measure.dead_additional_code)
+        else:
+            query &= Q(additional_code=None)
+
+        # matching_measures = (
+        #     type(measure)
+        #     .objects.filter(query)
+        #     .current()
+        #     .exclude(version_group=measure.version_group)
+        # )
```
Comment (NDevox) [inline measures/business_rules.py:345]: Delete commented code

File: measures/business_rules.py [hunk 2]
```diff
@@ -297,28 +302,47 @@ def validate(self, measure):
...
             measure_type__sid=measure.measure_type.sid,
             geographical_area__sid=measure.geographical_area.sid,
+            reduction=measure.reduction,
         )
         if measure.order_number is not None:
             query &= Q(order_number__sid=measure.order_number.sid)
+        elif measure.dead_order_number is not None:
+            query &= Q(dead_order_number=measure.dead_order_number)
+        else:
+            query &= Q(order_number=None)
```
Comment (NDevox) [inline measures/business_rules.py:331]: `Q(order_number__isnull=True, dead_order_number__isnull=True)`?

File: measures/business_rules.py [hunk 3]
```diff
@@ -297,28 +302,47 @@ def validate(self, measure):
...
+
         if measure.additional_code is not None:
             query &= Q(additional_code__sid=measure.additional_code.sid)
-        if measure.reduction is not None:
-            query &= Q(reduction=measure.reduction)
+        elif measure.dead_additional_code is not None:
+            query &= Q(dead_additional_code=measure.dead_additional_code)
+        else:
+            query &= Q(additional_code=None)
```
Comment (NDevox) [inline measures/business_rules.py:338]: `Q(additional_code__isnull=True, dead_additional_code__isnull=True)`?

**Review by NDevox — COMMENTED** [review]
Couple small points

**Review by stuaxo — APPROVED** [review]

**Pull request merged**